CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ cable_set_build_type(DEFAULT Release CONFIGURATION_TYPES Debug;Release;Coverage)
 
 project(fizzy LANGUAGES CXX)
 
+include(TestBigEndian)
+TEST_BIG_ENDIAN(IS_BIG_ENDIAN)
```

Comment:
```suggestion
test_big_endian(is_big_endian)
```

Reply:
This is a copy paste from Solidity and it works (checked with `NOT IS_BIG_ENDIAN`).

What is the difference between lower and upper case?

Reply:
No functional difference, just naming convention. CMake in general started using lower case for function names. Moreover, I prefer using lower case private variables.

Reply:
So are cmake variables/function names case insensitive or `TestBigEndian` exports both `test_big_endian` and `TEST_BIG_ENDIAN`?

Reply:
It is case insensitive but only for function/macro names to my understanding. Variable names are case sensitive.

Reply:
Thanks for the clarification. Updated both.